tdb/concatenate.py

```diff
@@ -0,0 +1,36 @@
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-f', '--files', nargs='*', default=[], help="tsvs that will be concatenated")
```

Comment:
Since both `output` and `files` are required, these don't need option flags. The `nargs` value for files should also be "+" since you want one or more inputs (as opposed to zero or more).

Another standard pattern for concatenating from an unknown number of inputs to a single output is to write the output to standard out instead of defining a specific output file. Then you can just have this one argument as follows:

```python
parser.add_argument('files', nargs='+', help="tsvs that will be concatenated")
```

And in the `concat` function you can just `print()` instead of writing to an output file.